proto/osmosis/twap/v1beta1/query.yml

```diff
@@ -3,18 +3,18 @@ keeper:
   struct: "Keeper"
 client_path: "github.com/osmosis-labs/osmosis/v12/x/twap/client"
 queries:
-  GetArithmeticTwap:
+  ArithmeticTwap:
     proto_wrapper:
       default_values:
         Req.end_time: "time.Now()"
       query_func: "k.GetArithmeticTwap"
     cli:
-      cmd: "GetArithmeticTwap"
-  GetArithmeticTwapToNow:
+      cmd: "ArithmeticTwap"
```

Comment:
I know this is unrelated to the core of the work in this PR but should we also change `time.Now()` above? to `ctx.BlockTime()`?

Reply:
100%